po/pt-BR.po

```diff
@@ -1929,17 +1882,17 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
-"```rust, editable\n"
-"fn main() { // Point de entrada do programa\n"
-"    let mut x: i32 = 6; // Vinculação de variável mutável\n"
-"    print!(\"{x}\"); // Macro para impressão, como printf\n"
-"    while x != 1 { // Sem parênteses em torno da expressão\n"
-"        if x % 2 == 0 { // Matemática como em outras linguagens\n"
-"            x = x/2;\n"
+"```rust,editable\n"
+"fn main() {              // Ponto de entrada do programa\n"
+"    let mut x: i32 = 6;  // Atribuição de uma variável mutável\n"
+"    print!(“{x}”);       // Macro para escrever na tela, como printf\n"
```

Comment:
These beautiful "typographic" quotes won't actually work. I'm waiting on #415 to be fixed upstream so that we could catch this.

Reply:
Done.